sdk/maps/maps-search/samples-dev/search.ts

```diff
@@ -0,0 +1,349 @@
+  console.log(await client.getGeometries([...geometryIds, "8bceafe8-3d98-4444-b29b-fd81d3e9adf5"]));
+
+  console.log(" --- Search along route:");
+  const searchALongRouteQuery = "burger";
+  const maxDetourTime = 1000;
+
+  const searchAlongRoute: GeoJsonLineString = {
+    type: "LineString",
+    coordinates: [
+      [-122.143035, 47.653536],
+      [-122.187164, 47.617556],
+      [-122.114981, 47.570599],
+      [-122.132756, 47.654009],
+    ],
+  };
+  const searchAlongRouteOptions = { top: 2 };
+  console.log(
+    await client.searchAlongRoute(
+      searchALongRouteQuery,
+      maxDetourTime,
+      searchAlongRoute,
+      searchAlongRouteOptions
+    )
+  );
+
+  console.log(" --- Search inside geometry: single polygon");
+  const searchInsideGeometryQuery = "burger";
+  const searchGeometryPolygon: GeoJsonPolygon = {
+    type: "Polygon",
+    coordinates: [
+      [
+        [-122.43576049804686, 37.7524152343544],
+        [-122.43301391601562, 37.70660472542312],
+        [-122.36434936523438, 37.712059855877314],
+        [-122.43576049804686, 37.7524152343544],
+      ],
+    ],
+  };
+  const searchInsideGeometryOptions = { top: 2 };
+  console.log(
+    await client.searchInsideGeometry(
+      searchInsideGeometryQuery,
+      searchGeometryPolygon,
+      searchInsideGeometryOptions
+    )
+  );
+
+  console.log(" --- Search inside geometry: multiple polygons");
+  const searchGeometryPolygons: GeoJsonPolygonCollection = {
+    type: "GeometryCollection",
+    geometries: [
+      {
+        type: "Polygon",
+        coordinates: [
+          [
+            [-122.43576049804686, 37.7524152343544],
+            [-122.43301391601562, 37.70660472542312],
+            [-122.36434936523438, 37.712059855877314],
+            [-122.43576049804686, 37.7524152343544],
+          ],
+        ],
+      },
+      {
+        type: "Polygon",
+        coordinates: [
+          [
+            [-121.43576049804686, 38.7524152343544],
+            [-121.43301391601562, 38.70660472542312],
+            [-121.36434936523438, 38.712059855877314],
+            [-121.43576049804686, 38.7524152343544],
+          ],
+        ],
+      },
+    ],
+  };
+  console.log(
+    await client.searchInsideGeometry(
+      searchInsideGeometryQuery,
+      searchGeometryPolygons,
+      searchInsideGeometryOptions
+    )
+  );
+
+  console.log(" --- Search inside geometry: circles and polygons");
+  const searchGeometryPolygonsOrCircles: GeoJsonCircleOrPolygonFeatureCollection = {
+    type: "FeatureCollection",
+    features: [
+      {
+        type: "Feature",
+        geometry: {
+          type: "Polygon",
+          coordinates: [
+            [
+              [-121.43576049804686, 38.7524152343544],
+              [-121.43301391601562, 38.70660472542312],
+              [-121.36434936523438, 38.712059855877314],
+              [-121.43576049804686, 38.7524152343544],
+            ],
+          ],
+        },
+        properties: {},
```

Comment:
`properties` can be removed since it's optional?

Reply:
Yes, thanks for pointing out!

Reply:
Fixed in da93ad9afc3806aadc463187d9cedff7a2f7f830